src/template.js

```diff
@@ -1,22 +1,61 @@
-<!DOCTYPE html>
+/* eslint-disable max-len */
+const path = require('path');
+const fs = require('fs');
+
+const _ = require('lodash');
+
+const projectRoot = path.resolve(__dirname, '..');
+const assetsRoot = path.join(projectRoot, 'public');
+
+/**
+ * Escapes `<` characters in JSON to safely use it in `<script>` tag.
+ */
+function escapeJson(json) {
+  return JSON.stringify(json).replace(/</gu, '\\u003c');
+}
+
+function getAssetContent(filename) {
+  const assetPath = path.join(assetsRoot, filename);
+
+  if (!assetPath.startsWith(assetsRoot)) {
+    throw new Error(`"${filename}" is outside of the assets root`);
+  }
+
+  return fs.readFileSync(assetPath, 'utf8');
+}
+
+function getScript(filename, mode) {
+  if (mode === 'static') {
+    return `<!-- ${_.escape(filename)} -->
+<script>${getAssetContent(filename)}</script>`;
+  } else {
+    return `<script src="${_.escape(filename)}"></script>`;
+  }
+}
+
+function renderViewer({title, enableWebSocket, chartData, defaultSizes, mode} = {}) {
+  return `<!DOCTYPE html>
```

Comment:
I see a few drawbacks of this approach:
- No syntax highlighting
- Much more difficult to do conditional branching e.g. `if/else` if we'll need it in the future.
- You have to remember to call `_.escape` manually

Reply:
I'm not saying that I'm strongly against this PR, no. We can live with these drawbacks, but only if it brings some significant improvements. That's why I asked to provide difference in `node_modules` sizes.

Reply:
- Syntax highlighting could be added in some editors/plugins by adding a "fake" tag `html`. I'll add that as a commit so you can see what it looks like.
- Agreed that conditionals are more cumbersome but do you expect the templates to get significantly more complex than they are today?
- This is true (though you also have to be mindful of `<%=` vs `<%-` in EJS) and I wouldn't do it for a large server rendered app but as of today there's just two variables in the templates here that need to be escaped.

To me, this is trade-off of complexity. EJS is certainly nicer than template literals but for a template this small it's, IMHO, not worth the cost of the dependencies. Some of which are very old and don't even deduplicate.